mrc/localization/color/kmeans/analyser.py

```diff
@@ -6,18 +6,55 @@
 
 
 class Predicates:
```

Comment:
It's not a very beautiful solution. We could consider changing it.

Reply:
We could, but until we figure out something nicer let's leave it at that